rest/test/db/CatapultDb_spec.js

```diff
@@ -695,23 +696,63 @@ describe('catapult db', () => {
 		const addTransactionAtHeight = (height, type) => {
 			const aggregateId = test.db.createObjectId(id);
 			const hash = new Binary(Buffer.from(createTransactionHash(id++)));
-			const meta = { height, hash, addresses: [] };
+			const meta = { height: Long.fromNumber(height), hash, addresses: [] };
 			transactions.push({ _id: aggregateId, meta, transaction: { type } });
 
 			const numDependentDocuments = (options || {}).numDependentDocuments || 0;
-			for (let j = 0; j < numDependentDocuments; ++j)
-				transactions.push({ _id: test.db.createObjectId(id++), meta: { height, aggregateId }, transaction: {} });
+			for (let j = 0; j < numDependentDocuments; ++j) {
+				transactions.push({
+					_id: test.db.createObjectId(id++),
+					meta: { height: Long.fromNumber(height), aggregateId },
+					transaction: {}
+				});
+			}
 		};
 
 		for (let i = 0; i < numTransactionsPerHeight; ++i) {
-			// transactionsAtHeight only worked for both aggregates, so pick each type alteratively
+			// transactionsAtHeight only worked for both aggregates, so pick each type alternatively
 			const type = 0 === i % 2 ? EntityType.aggregateComplete : EntityType.aggregateBonded;
 			heights.forEach(height => { addTransactionAtHeight(height, type); });
 		}
 
 		return transactions;
 	};
 
+	/**
+	 * It creates the blocks for the given transactions.
+	 *
+	 * @param {object[]} transactions the transactions to know the required blocks
+	 * @returns {object[]} the block of the transactions to be stored
+	 */
```

Comment:
generally, we don't need to fully document private helper functions

Reply:
Cool, public `addBlockMetaToEntityList` has the doc as it's used externally too